### DISCUSSION TITLE
八月更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Wfa.Uwp#41] [Announcements]
## 🚀 功能

- 支持物品订单的Mod等级显示及排序 @Richasy (#39)
- 将社区数据的默认自动更新改为默认手动更新 @Richasy (#38)

## 🐛 Bug 修复

- 修复一些简单的布局与提示问题 @Richasy (#40)
- 改变扎里曼号的状态图标 @Richasy (#37)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Wfa.Uwp/releases/tag/v6.2208.1.0'>八月更新</a>.</em>

**Comment by qunianfengxue** [comment]
没有紫卡计算器吗？

**Comment by JWhiteCat** [comment]
最新版改不了中文

**Reply by JWhiteCat** [reply]
修改系统首选语言后可以了

**Comment by Te-soro** [comment]
虚空裂缝标记钢铁

**Comment by sos02** [comment]
铁子啊，你能不能把网络搞定一下啊，你不要再研发什么乱七八糟的东西，你能把这个东西攻克好了就行了，真的，你那个网络啊，阿西吧，你教我们怎么翻墙，还是你自己内置你那个工具箱里面，什么都没有，更新不出来的吗，卧槽，发现你变懒了，以前挺勤快的
